oauthenticator/github.py

```diff
@@ -27,6 +28,30 @@
 else:
     GITHUB_API = '%s/api/v3/user' % GITHUB_HOST
 
+
+def _api_headers(access_token):
+    return {"Accept": "application/json",
+            "User-Agent": "JupyterHub",
+            "Authorization": "token {}".format(access_token)
+           }
+
+
+def _get_next_page(response):
+    # Github uses Link headers for pagination.
+    # See https://developer.github.com/v3/#pagination
+    link_header = response.headers.get('Link')
+    if not link_header:
+        return
+    # parse the Link header; is this regex general enough?
+    link_regex = r',? *< *(.*?) *> *; *rel="(.*?)"'
```

Comment:
Isn't it preferable to just import it from `requests`, given that it is a dependency of `jupyterhub`? This is what I have done for now. If vendoring is the preferable solution in this case, then I will do this